uint8 ADD = 0
uint8 DELETE = 1

uint8 action

int32 width
int32 height
int32 left
int32 top
std_msgs/ColorRGBA bg_color

int32 line_width
float32 text_size
string font
std_msgs/ColorRGBA fg_color

string text
